### DISCUSSION TITLE
Can't load YouTube channel details

### DISCUSSION THREAD

**Discussion by CelianGdfrd** [libre-tube/LibreTube#6804] [General]
### Steps to reproduce

Click on a YouTube channel image to load the channel page.

### Expected behavior

Channel should show, with informations, last videos, playlists...

### Actual behavior

Loading indefinitely and not showing the page.

### LibreTube version

0.26.0

### Android version

Android 14

### Other details

Happens since 0.26.0 update, didn't occur before.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Can't reproduce, it's likely just the instance you use that's blocked on that specific channel API route.
